### ISSUE TITLE
About com.clxcommunications.xms.ConcurrentException: java.io.IOException: Connection reset by peer

### ISSUE THREAD

**Issue by uugan** [sinch/sinch-java-sms#3]
```java
sinchConnection = ApiConnection.builder()
                    .servicePlanId("*****")
                    .token("******")
                    .start();
```
and used this global variable as like below
```java
  MtBatchTextSmsResult result = _sinchConnection.createBatch(ClxApi.batchTextSms()
                                                                                   .sender("****")
                                                                                   .addRecipient(msisdn)
                                                                                   .body(sms)
                                                                                   .build());
```
but here gives error on first call like:
```
com.clxcommunications.xms.ConcurrentException: java.io.IOException: Connection reset by peer
        at com.clxcommunications.xms.Utils.unwrapExecutionException(Utils.java:138)
        at com.clxcommunications.xms.ApiConnection.createBatch(ApiConnection.java:586)
```
But after second call this function works normally. There should be connection checker in createBatch method.

**Comment by thomasf147** [comment]
This will be fixed in the next release. Master code now contains a fix.

**Comment by thomasf147** [comment]
Fixed in release 1.0.4
